linchpin/shell/__init__.py

```diff
@@ -127,7 +127,7 @@ def _handle_results(ctx, results, return_code):
               help='Enable verbose output')
 @click.option('--version', is_flag=True,
               help='Prints the version and exits')
-@click.option('--creds-path', type=click.Path(), envvar='CREDS_PATH',
+@click.option('--creds-path',default= "./",type=click.Path(), envvar='CREDS_PATH',
```

Comment:
I think this might not be a correct implementation of defaulting the credential path 
you should change 
https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/linchpin/linchpin.constants#L62 
from 
```
default_credentials_path = %(default_config_path)s
```
to 
```
default_credentials_path = {{ workspace }}/
```
If you want to search the workspace by default 
such that users will still have a way to configure the credentials path.

Reply:
You can still configure the credential path with my way
You can still do (e.x. linchpin --cred-path ./credential)